zipline/pipeline/loaders/synthetic.py

```diff
@@ -250,6 +251,9 @@ def make_bar_data(asset_info, calendar):
         DataFrame with asset_id as index and 'start_date'/'end_date' columns.
     calendar : pd.DatetimeIndex
         The trading calendar to use.
+    holes : dict[int -> tuple[pd.Timestamps]]
```

Comment:
We should add a note here that this is optional and that the default is no holes.